fraudserv/service_test.go

```diff
@@ -26,6 +27,9 @@ func TestService_SubscribeBroadcastValid(t *testing.T) {
 	require.NoError(t, serv.Start(ctx))
 
 	fraud := fraudtest.NewValidProof()
+	serv.AddVerifier(fraud.Type(), func(fraudProof gofraud.Proof) (bool, error) {
+		return true, nil
+	})
```

Comment:
If it's optional, why do we add this? If we want to test the AddVerifier, it's better its own test. WDYT?

Reply:
moved to a separate test.